### PR TITLE
kelvin commit

### PR DESCRIPTION
**Pull request by kelvingevariya**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (keval710) [inline .gitignore:1]: jkbi

Comment (kelvingevariya) [reply .gitignore:1]: hii

File: index.ts [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+
+
+const app = express();
+
+const PORT: number = 4000;
+
+app.use(express.json());
+
+app.get("/scores", (req: Request, res: Response) => {
```
Comment (keval710) [inline index.ts]: make sure the request is POST

Comment (kelvingevariya) [reply index.ts]: Sure

File: index.ts [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    const topic: [] = req.body.topic;
+    const format: [] = req.body.format;
+    const source: [] = req.body.source;
+
+
+
+    //* Adding the initial scores to all sections
+    function calculatePoints() {
```
Comment (keval710) [inline index.ts:28]: add an interface to the respective function parameter

File: index.ts [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+        responseJSON.forEach((sec) => {
```
Comment (keval710) [inline index.ts]: add an interface or Type allies

File: index.ts [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+                });
+                i++;
+
+            }
+
+        });
+
+        for (let i = 0; i < topic.length; i++) {
+            //! Quiz in topic 
+            let pointsAddedQuiz = [];
```
Comment (keval710) [inline index.ts]: assign variable type

File: index.ts [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+                        responseJSON.forEach((secName) => {
+                            if (secName.section_type === "Quiz") {
+                                //@ts-ignore
+                                secName.points += 10;
+                            }
+                        });
+                    }
+                    //! Condition for news
+                    if (elementType === "Reading") {
+                        responseJSON.forEach((secName) => {
```
Comment (keval710) [inline index.ts]: assign interface or type

File: index.ts [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+                            if (secName.section_type === "news") {
+                                //@ts-ignore
```
Comment (keval710) [inline index.ts]: remove all ts-ignore

File: index.ts [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+
+
+const app = express();
+
+const PORT: number = 4000;
+
+app.use(express.json());
+
+app.get("/scores", (req: Request, res: Response) => {
+    const topic: [] = req.body.topic;
```
Comment (keval710) [inline index.ts]: assign type of array

File: index.ts [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+                                if (idFormat.topic_id === topic[j]) {
+                                    //@ts-ignore
+                                    secName.points += 100;
+                                }
+                            }
+                        })
+                    }
+                })
+            }
+        }
```
Comment (keval710) [inline index.ts:123]: add comment for closing function

**Pull request merged**